standard-integration/public/index.html

```diff
@@ -42,8 +51,7 @@
                   orderData,
                   JSON.stringify(orderData, null, 2)
                 );
-                var transaction =
-                  orderData.purchase_units[0].payments.captures[0];
+                var transaction = orderData.purchase_units[0].payments.captures[0];
```

Comment:
should we declare `transaction` as  `const` instead of `var`?

Reply:
Yes we should standardize on `const`. We can fix that in a future PR.

Reply:
I just realized we already use `const` in our public docs so I made the same change here. Great call out!